server.py

```diff
@@ -191,6 +191,42 @@ def cleanup_usb_assets(media_dir='/media'):
                         assets_helper.delete(conn, asset['asset_id'])
 
 
+@celery.task
+def display_status():
+    """Request power status of primary CEC device."""
+    cmd = sh.Command('/usr/bin/cec-client')
```

Comment:
Can be done so `sh.Command('/usr/bin/cec-client')(sh.echo('pow 0'), '-d', '1', '-s')`